fe/fe-core/src/main/java/com/starrocks/transaction/LakeTableTxnLogApplier.java

```diff
@@ -105,7 +105,7 @@ public void applyVisibleLog(TransactionState txnState, TableCommitInfo commitInf
 
     public void applyVisibleLogBatch(TransactionStateBatch txnStateBatch, Database db) {
         for (TransactionState txnState : txnStateBatch.getTransactionStates()) {
-            TableCommitInfo tableCommitInfo = txnState.getTableCommitInfo(txnStateBatch.getTableId());
+            TableCommitInfo tableCommitInfo = txnState.getTableCommitInfo(txnStateBatch.getTableId().get(0));
```

Comment:
The transaction here is probably a multi-table transaction, right? If so, shouldn't it get all the TableCommitInfo and process it? Why is only the first table being processed here?

Reply:
ditto